topics/dot-net/index.md

```diff
@@ -0,0 +1,15 @@
+---
+created_by: Microsoft
+display_name: .NET
+logo: dot-net.png
+released: February 13, 2002
+short_description: .NET Framework is used to create desktop applications and Web services.
+topic: dot-net
+related: software, framework
+url: https://microsoft.com/net
+wikipedia_url: https://en.wikipedia.org/wiki/.NET_Framework
+---
+The .NET Framework consists of the common language runtime (CLR) and the .NET Framework class library. It provides a consistent object-oriented programming environment and used to develop console apps, Windows GUI apps, Windows services etc.
```

Comment:
- `it provides ... and ...` implies that you're going to use  another`<verb>` with `s` at the end instead of `ed`. To make this sentence correct, remove the `and`.
- Personal preference: `apps` should be `applications` instead since IMO "app" is used more in the mobile space and "application" is used more when talking about applications in general.
- Change `, Windows services etc.` to `, and Windows services`.